### ISSUE TITLE
Rename the platform into Web Platform

### ISSUE THREAD

**Issue by hamzaed** [xatkit-bot-platform/xatkit-react-platform#8]
I think this platform should be called `WebPlatform` instead of `ReactPlatform`. Indeed, React is the technology used to implement the chat widget but still I think we should use a general form to describe the platform (it provides a Web widget to communicate with Xatkit background).

**Comment by gdaniel** [comment]
Agreed, already discussed with @jcabot and we came to the same conclusion (and same name!)
